Makefile

```diff
@@ -35,6 +35,14 @@ run-with-external-etcd:
 	hack/start-multicluster-controlplane.sh false
 .PHONY: run-with-external-etcd
 
+run-auto-generate-cert:
```

Comment:
what do you mean about auto-generate cert? auto-generate all certs, including ca / serve certs / client certs, correct?

My expectation is to get the configuration information from a configmap. the configmap data can be
```
config.yaml
   apiServer:
      subjectAltNames:
      - "xxx"
      ca: xxx/
```
the configmap can mount to /etc/controlplane/
if the customers configure the ca, then we use this customized ca to generate certs. if no, we use self-signed ca to generate certs.